Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where ProcessID in (295,428)| where FileName !contains \"PSMSessions\"| where FileName !contains \"PVWAConfig\"| where FileName !contains \"PasswordManagerShared\"| where FileName !contains \"VaultInternal\"| where FileName !contains \"PasswordManager\"| where FileName !contains \"PVWAPrivateUserPrefs\"| where FileName !contains \"ConjurSync\"| where FileName !contains \"SharedAuth_Internal\"| where FileName !contains \"PSM\"| where SourceUserName !contains \"PasswordManager\"| summarize count() by FileName",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0
+      },
+      "name": "Successful CPM operations"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where DeviceAction contains \"disable\"| summarize count() by FileName, DestinationUserName, OldFileID",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0
+      },
+      "name": "Account objects accessed"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where ProcessID in (295,428)| where FileName !contains \"PSMSessions\"| where FileName !contains \"PVWAConfig\"| where FileName !contains \"PasswordManagerShared\"| where FileName !contains \"VaultInternal\"| where FileName !contains \"PasswordManager\"| where FileName !contains \"PVWAPrivateUserPrefs\"| where FileName !contains \"ConjurSync\"| where FileName !contains \"SharedAuth_Internal\"| where FileName !contains \"PSM\"| where SourceUserName !contains \"PasswordManager\"| summarize count() by SourceUserName, TimeGenerated",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0
+      },
+      "name": "Users accessing accounts"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where ProcessID in (359,360,361,412,411) | summarize audit=makeset(OldFileID) by ExternalID, DestinationUserName, SourceUserName",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0
+      },
+      "name": "Audit information"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where ProcessID in (295,428)| where FileName contains \"ConjurSync\"| where SourceUserName contains \"Sync_components\"| summarize AggregatedValue = count() by bin(TimeGenerated, 1h)| sort by TimeGenerated desc| render timechart",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0
+      },
+      "name": "Conjur Vault syncs"
+    }
+  ],
+  "workbookKey": "CyberArkWorkbook",
```

Comment:
Remove the following metadata block from this workbook json file. This needs to go in the workbooksMetadata.json file as per instructions provided @ https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks#how-to-contribute-new-workbook 

{
  "workbookKey": "CyberArkWorkbook",
  "logoFileName": "CyberArk_Logo.svg",
  "description": "The CyberArk Syslog connector allows you to easily connect all your CyberArk security solution logs with your Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. Integration between CyberArk and Azure Sentinel makes use of the CEF Data Connector to properly parse and display CyberArk Syslog messages.",
  "dataTypesDependencies": [ "CommonSecurityLog" ],
  "dataConnectorsDependencies": [ "CyberArk" ],
  "previewImagesFileNames": [ "CyberArk_White1.png", "CyberArk_Black1.png" ],
  "version": "1.1",
  "title": "CyberArk EPV Events",
  "templateRelativePath": "CyberArkEPV.json",
  "subtitle": "",
  "provider": "Microsoft",
  "fromTemplateId": "CyberArkWorkbook",
  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
}